lib/tictactoe.rb

```diff
@@ -1,42 +1,78 @@
+require 'ai_basic'
+require 'ai_minimax'
 require 'game_board'
 require 'game_rules'
+require 'game_state'
 require 'player_manager'
-require 'game_loop'
-require 'game_screen'
-class TicTacToe
 
-  attr_accessor :board, :game_loop
+class TicTacToe
 
-  def initialize (game_io, player1, player2)
+  def initialize
```

Comment:
Mike, 
Yes, my thought was that the user of the gem should just be able to create a TicTacToe game and not have to wire parts together. However, the dependencies passed in that were removed didn't seem to apply as much to the game if it was no longer a terminal application. IE the game_io should be handled by the application using the gem. Also I removed the player classes as the symbols used for players would no longer need to be changeable. I see no reason to have the internal board change what symbols are used for player1 and player2. My thought was that a web app or any other application using this gem would decide for its self how to display the board. ie if the value of the space is :X the web app would say okay put the picture or character I use for :X in that spot.